.travis.yml

```diff
@@ -21,12 +21,13 @@ jobs:
     - { stage: python3working, python: 3.6, env: TOXENV=codestyle}
     - { stage: python3working, python: 3.6, env: TOXENV=docs }
     - { stage: python3working, python: 3.6, env: TOXENV=assets }
+    - { stage: python3, python: 3.6, env: TOXENV=users }
```

Comment:
You need to replicate that split to the python 2 jobs as well...

Reply:
https://github.com/mozilla/addons-server/pull/10450/commits/6ddf8f0753c14655f1ebc311104aa806fa150961